force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -238,11 +278,11 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Set<String> createStringSet(List<String> inputList) {
         // Create a new Set of strings
-        Set<String> stringSet = new Set<String>();
+        Set<String> stringSet = new Set<String>(inputList);
 
         //add strings to the set
 
-        return null; // Replace null with the variable you used to store the result
+        return stringSet; // Replace null with the variable you used to store the result
```

Comment:
Nailed it!  See my comment above about just returning the value and skipping the variable assignment. 

Love that you solved this without iteration.